airbyte-integrations/connectors/source-stripe/Dockerfile

```diff
@@ -12,5 +12,5 @@ COPY main.py ./
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=2.0.0
+LABEL io.airbyte.version=2.0.1
```

Comment:
Should this be a major version bump per @davydov-d's comment?

Reply:
updated